enterprise-suite/scripts/lbc.py

```diff
@@ -76,10 +79,6 @@ def fail(msg):
     sys.exit(msg)
 
 
-def make_tempdir():
```

Comment:
Some tests override this function, they will have to be changed if you remove it

Reply:
Yes discovered this when the tests failed...

Reply:
Replaced with `make_fetchdir` farther down in the file.